apps/settings/tests/SetupChecks/DataDirectoryProtectedTest.php

```diff
@@ -37,17 +37,17 @@
 use Test\TestCase;
 
 class DataDirectoryProtectedTest extends TestCase {
-	private IL10N|MockObject $l10n;
-	private IConfig|MockObject $config;
-	private IURLGenerator|MockObject $urlGenerator;
-	private IClientService|MockObject $clientService;
-	private LoggerInterface|MockObject $logger;
-	private DataDirectoryProtected|MockObject $setupcheck;
+	private IL10N&MockObject $l10n;
```

Comment:
```suggestion
	private IL10N|MockObject $l10n;
```

For PHP 8.0 :disappointed:

Reply:
Sadly yes :(